Article/Article.js

```diff
@@ -112,3 +112,31 @@ const data = [
   Step 5: Add a new article to the array. Make sure it is in the same format as the others. Refresh the page to see the new artible
 
 */
+
```

Comment:
This would be a good spot to add your loop so we can gather all the data -
```
data.forEach( data => {
    articles.appendChild(creator(data.title, data.date, data.firstParagraph, data.secondParagraph, data.thirdParagraph)
}); 

``` 

This will take all of the data above and push it into your function below.